environments/puppet/hiera.yaml

```diff
@@ -35,4 +35,7 @@ hierarchy:
   - name: SIMP specific data - Please do not modify
     paths:
     - "simp_config_settings.yaml"
+    - "scenarios/%{facts.os.family}/%{::simp_scenario}.yaml"
+    - "scenarios/%{facts.os.name}/%{facts.os.release.full}/%{::simp_scenario}.yaml"
```

Comment:
I don't think that these should be under `%{::simp_scenario}`. Instead, I think that they should just be one level up as YAML files for that version.

The reason is that this simply will not work on EL8, regardless of the scenario and it defaults it back to what is the module default anyway.

I'm tempted to say that it should actually be at the top of the stack completely since it's just not supported on that OS.